src/content/docs/ru/tutorials/add-content-collections.mdx

```diff
@@ -0,0 +1,389 @@
+---
+title: Учебное пособие - Расширение с помощью коллекций контента
+description: >-
+  Преобразуйте код учебника Создать блог с маршрутизации на основе файлов на коллекции контента.
+i18nReady: true
+---
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro';
+import Box from '~/components/tutorial/Box.astro';
+import MultipleChoice from '~/components/tutorial/MultipleChoice.astro';
+import PreCheck from '~/components/tutorial/PreCheck.astro';
+import Option from '~/components/tutorial/Option.astro';
+
+**Коллекции контента** - это мощный способ управления группами схожего контента, например, записями в блоге. Коллекции помогают упорядочить документы, проверить YAML frontmatter и обеспечить автоматическую безопасность типов TypeScript для всего контента (даже если вы сами не пишете на TypeScript).
+
+<PreCheck>
+  - Переместите папку с записями блога в `src/content/`
```

Comment:
Because the PreCheck block starts with word "Приготовьтесь..", we have to change wording a bit.
```suggestion
  - Переместить папку с записями блога в `src/content/`
  - Создать схему для определения frontmatter записей блога
  - Использовать `getCollection()` для получения контента и метаданных записей блога
```